subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts

```
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
     }
 }
 // end::violation-rules-configuration[]
+
+// tag::testtask-configuration-defaults[]
+tasks.getByName<Test>("test") {
+    extensions.configure(JacocoTaskExtension::class) {
```

:x: should use `configure<T> {}`, there's another instance of that above line 27
```suggestion
    configure<JacocoTaskExtension> {
```